cmd/buildkitd/config/gcpolicy.go

```diff
@@ -104,3 +104,19 @@ func stripQuotes(s string) string {
 	}
 	return s
 }
+
+func DetectDefaultGCCap() DiskSpace {
+	return DiskSpace{Percentage: DiskSpacePercentage}
+}
+
+func (d DiskSpace) AsBytes(root string) int64 {
+	if d.Bytes != 0 {
+		return d.Bytes
+	}
+	if d.Percentage == 0 {
+		return 0
+	}
+
+	diskSize := getDiskSize(root)
```

Comment:
The rounding seems to be lost

Reply:
thanks for the explanation, getting it back.